test/setup.ts

```diff
@@ -1,4 +1,7 @@
 import { act } from 'react-dom/test-utils';
+import { configure } from '@testing-library/react';
+
+configure({ throwSuggestions: true });
```

Comment:
>Duration  67.79s (transform 1.28s, setup 7.69s, collect 7.75s, tests 29.41s, environment 12.71s, prepare 2.69s)

The tests take over 1 minute now, I think it's fine to try it from time to time to improve tests, but I'm 👎 on keeping this enabled.

Reply:
How long does it take without this change?

Reply:
NVM I thought the GH CI was faster than that

Reply:
We can add it back